src/extensions/codemirror/CodeMirror.tsx

```diff
@@ -1,5 +1,5 @@
-import React, { TextareaHTMLAttributes, useEffect, useRef } from "react";
-import CodeMirror, { ModeSpec, ModeSpecOptions } from "codemirror";
+import React, {TextareaHTMLAttributes, useEffect, useRef} from "react";
```

Comment:
@andreas-schultz is it possible that different linting rules are used here? Otherwise there shouldn't be changes?